docs/example-scenario/data/hybrid-etl-with-adf.md

```diff
@@ -0,0 +1,102 @@
+
+The decision is much easier when you already have existing SSIS packages that have on-premise dependencies such as data sources, or files which are not accessible from Azure.  In this scenario, your only option is Self-hosted IR. This approach provides the most flexibility to leverage the cloud as the orchestration engine, without having to rewrite existing packages.
+
+Ultimately, the intent is to move the processed data into the cloud for further refinement or to combine with other data stored in the cloud.  As part of the design process, keep track of the number of activities used in the ADF pipelines.  You can learn more about [pipelines and activities](https://docs.microsoft.com/en-us/azure/data-factory/concepts-pipelines-activities).
+
+### Availability, Scalability, and Security
+
+> How do I need to think about managing, maintaining, and monitoring this long term?
+
+> Are there any size considerations around this specific solution?  
+> What scale does this work at?  
+> At what point do things break or not make sense for this architecture?
+
+> Are there any security considerations (past the typical) that I should know about this?
+
+## Pricing
+
+Azure Data Factory is a very cost-effective way to orchestrate data movement in the cloud.  The cost is based on the several factors.
+
+* Number of pipelines executions
+* Number of entities/activities used within the pipeline
+* Number of monitoring operations
+* Number of Integration Runs (Azure-hosted IR or Self-hosted IR)
+
+ADF uses a consumption-based billing; therefore, cost is only incurred during pipeline executions and monitoring.  The execution of a basic pipeline would cost as little as $0.50 cents and the monitoring as little as $25 cents. The [Azure cost calculator](https://azure.microsoft.com/en-us/pricing/calculator/) can be used to create a more accurate estimate based on your specific work load.
+
+When running a hybrid ETL workload, you must factor in the cost of the Virtual machine used to host your SSIS packages. This is explained in the above considerations section of this document. This cost is based on the size of the VM ranging from a D1v2 (1 core, 3.5 GB RAM, 50GB Disk) to E64V3 (64 cores, 432GB RAM, 1600GB Disk).  If you need further guidance on selection the appropriate VM size refer to [VM performance considerations](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs#performance-considerations).
+
+## Next Steps
+
+* To learn more about [Azure Data Factory](https://azure.microsoft.com/en-us/services/data-factory/)
+* Get started with Azure Data Factory by following the [Step-by-Step tutorial](https://docs.microsoft.com/en-us/azure/data-factory/#step-by-step-tutorials)
+* [Provision the Azure-SSIS Integration Runtime in Azure Data Factory](https://docs.microsoft.com/en-us/azure/data-factory/tutorial-deploy-ssis-packages-azure)
```

Comment:
Consider add a link to our [ADF reference architecture](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/data/enterprise-bi-adf) for an example that uses Analysis Service in Azure